integration_tests/testprovider/TestProvider.cs

```diff
@@ -46,40 +46,40 @@ public override Task<ParameterizeResponse> Parameterize(ParameterizeRequest requ
 
     public override Task<GetSchemaResponse> GetSchema(GetSchemaRequest request, CancellationToken ct)
     {
-        var schema = """
```

Comment:
Raw strings `"""..."""` were introduced in C# 11, however .NET 6.0 only supports up to C# 10. Convert this to use verbatim strings `@"..."`, which unfortunately require double quoting quotes.